package.json

```diff
@@ -23,25 +26,32 @@
   },
   "homepage": "https://test-aaf.herokuapp.com/",
   "dependencies": {
+    "bcrypt": "^2.0.1",
+    "body-parser": "^1.18.2",
     "express": "^4.16.1",
+    "mongoose": "^4.13.11",
     "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
     "react-router-dom": "^4.2.2",
     "react-tap-event-plugin": "^3.0.2"
   },
   "devDependencies": {
-    "concurrently": "^3.5.0",
-    "enzyme": "^3.0.0",
-    "enzyme-adapter-react-16": "^1.0.0",
-    "enzyme-to-json": "^3.0.1",
-    "eslint": "^4.7.2",
-    "eslint-plugin-react": "^7.4.0",
+    "concurrently": "^3.5.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
+    "eslint": "^4.17.0",
+    "eslint-plugin-react": "^7.6.1",
+    "jest": "^22.2.2",
+    "jest-cli": "^22.2.2",
     "node-sass-chokidar": "0.0.3",
```

Comment:
I think we need to update this to a newer version. Node sass failing to build with node-gyp

Reply:
yeah i was about to push up a change to travis file to ignore node latest @jwu910